i18n/stdm_ne.ts

```diff
@@ -0,0 +1,7103 @@
+        <location filename="../data/configuration/config_updater.py" line="263"/>
+        <source>Error in creating {0} foreign key constraint.</source>
+        <translation>خطأ في إنشاء قيد المفتاح الأجنبي {0}</translation>
+    </message>
+    <message>
+        <location filename="../data/configuration/config_updater.py" line="267"/>
+        <source>{0} foreign key constraint successfully created.</source>
+        <translation> تم إنشاء قيد المفتاح الأجنبي {0} بنجاح</translation>
+    </message>
+    <message>
+        <location filename="../data/configuration/config_updater.py" line="276"/>
+        <source>creating</source>
+        <translation>انشاء</translation>
+    </message>
+    <message>
+        <location filename="../data/configuration/config_updater.py" line="279"/>
+        <source>updating</source>
+        <translation>تحديث</translation>
+    </message>
+    <message>
+        <location filename="../data/configuration/config_updater.py" line="282"/>
+        <source>deleting</source>
+        <translation>حذف</translation>
+    </message>
+    <message>
+        <location filename="../data/configuration/config_updater.py" line="228"/>
+        <source>entity</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>ContentAuthorizationToolbarAction</name>
+    <message>
+        <location filename="../plugin.py" line="819"/>
+        <source>Content Authorization</source>
+        <translation>تفويض المحتوى</translation>
+    </message>
+</context>
+<context>
+    <name>CoordinatesWidget</name>
+    <message>
+        <location filename="../ui/customcontrols/coordinates_editor.py" line="42"/>
+        <source>Y-Coordinate</source>
+        <translation>الاحداثيات الصادية</translation>
+    </message>
+    <message>
+        <location filename="../ui/customcontrols/coordinates_editor.py" line="46"/>
+        <source>X-Coordinate</source>
+        <translation>الاحداثيات السينية</translation>
+    </message>
+</context>
+<context>
+    <name>CopyEditor</name>
+    <message>
+        <location filename="../ui/wizard/copy_editor.py" line="105"/>
+        <source>Please enter a profile name.</source>
+        <translation>الرجاء إدخال اسم الملف الشخصي</translation>
+    </message>
+    <message>
+        <location filename="../ui/wizard/copy_editor.py" line="114"/>
+        <source>Entered name is already in use. Please enter another profile name.</source>
+        <translation>الاسم المدخل مستخدم سابقا. الرجاء ادخال اسم آخر للملف الشخصي</translation>
+    </message>
+</context>
+<context>
+    <name>DTimeProperty</name>
+    <message>
+        <location filename="../ui/wizard/ui_dtime_property.ui" line="14"/>
+        <source>Datetime properties</source>
+        <translation>خصائص الوقت والتاريخ</translation>
+    </message>
+    <message>
+        <location filename="../ui/wizard/ui_dtime_property.ui" line="20"/>
+        <source>Minimum</source>
+        <
```

Comment:
left out translation from Arabic. need to clean up before merging in trunk @neogeomat @wondie

Reply:
That is right @biplovbhandari For STDM translation, we use Transifex ( https://www.transifex.com/stdm/stdm/dashboard/  ). We have already reached 70%+. However, for the Nepali translation to be incorporated on STDM, the translation of QGIS Desktop should increase at least to 20% ( https://www.transifex.com/qgis/QGIS/ ). Your contribution to both projects is much welcome! @neogeomat has already joined both projects.

Reply:
Thanks for the instruction @wondie. I will look into that. Hope the training went well!

Reply:
@biplovbhandari it went well! I will actually share a document on how to do it, which is for non-programmers. 
@neogeomat could you please send me a test email to my email wondim81@gmail.com I will send you the instruction too.